gui/GameConqueror.xml

```diff
@@ -866,97 +953,11 @@ Special thanks: Igor Gnatenko, Andrea Stacchiotti, Mattias Muenster, Zhang Zhe</
-                <property name="visible">True</property>
-                <property name="can_focus">True</property>
-                <property name="receives_default">True</property>
-                <property name="use_underline">True</property>
-              </object>
-              <packing>
-                <property name="expand">False</property>
-                <property name="fill">False</property>
-                <property name="position">1</property>
-              </packing>
-            </child>
-          </object>
-          <packing>
-            <property name="expand">False</property>
-            <property name="fill">True</property>
-            <property name="pack_type">end</property>
-            <property name="position">2</property>
-          </packing>
-        </child>
       </object>
     </child>
     <action-widgets>
       <action-widget response="-5">ProcessListDialog_OK_Button</action-widget>
       <action-widget response="-6">ProcessListDialog_Cancel_Button</action-widget>
     </action-widgets>
   </object>
-  <object class="GtkImage" id="add_image">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
-    <property name="icon-name">list-add</property>
-  </object>
-  <object class="GtkImage" id="delete_image">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
-    <property name="icon-name">edit-delete</property>
-  </object>
-  <object class="GtkImage" id="edit_image">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
-    <property name="icon-name">gtk-edit</property>
-  </object>
-  <object class="GtkImage" id="load_image">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
-    <property name="icon-name">document-open</property>
-  </object>
-  <object class="GtkImage" id="save_image">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
-    <property name="icon-name">document-save</property>
-  </object>
-  <object class="GtkImage" id="find_image">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
-    <property name="icon-name">edit-find</property>
-  </object>
-  <object class="GtkImage" id="jumpto_image">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
-    <property name="icon-name">gtk-jump-to</property>
-  </object>
-  <object class="GtkImage" id="refresh_image">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
-    <property name="icon-name">view-refresh</property>
-  </object>
-  <object class="GtkImage" id="system_image">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
-    <property name="icon_name">computer</property>
-  </object>
```

Comment:
Reviewed by doing the same with glade 3.18. Same diff. Also checked manually. Merged. Thanks!